End2End/README.md

```diff
@@ -0,0 +1,13 @@
+# Microsoft Azure Functions in Java
+
+This repo contains several Java Azure Functions annotation samples and without annotation samples for different events.
+
+## Prerequisites
+- JDK 1.8
+- Maven 3.0+
+- [.NET Core SDK](https://www.microsoft.com/net/learn/get-started/windows)
+- [Azure Functions Core Tools v2](https://docs.microsoft.com/en-us/azure/azure-functions/functions-run-local#v2)
+- [Maven Plugin for Azure Functions](https://github.com/Microsoft/azure-maven-plugins/blob/develop/azure-functions-maven-plugin/README.md)
+
+## Azure Functions List and Test Pass
```

Comment:
Add instructions on how to run the tests. For example, login to azure and manually create the resources needed for the tests like storage account etc.
- Please list all the azure resources that need to be created
- Please list all the environment variables that need to be set to run them locally.

Reply:
Yes, I will list that.